membership-attribute-service/app/models/Attributes.scala

```diff
@@ -12,60 +10,87 @@ import play.api.mvc.Results.Ok
 
 import scala.language.implicitConversions
 import scalaz.syntax.std.boolean._
+import json._
 
 case class ContentAccess(member: Boolean, paidMember: Boolean, recurringContributor: Boolean, digitalPack: Boolean)
 
 object ContentAccess {
   implicit val jsWrite = Json.writes[ContentAccess]
 }
 
-case class Attributes(
-  UserId: String,
-  Tier: Option[String] = None,
-  MembershipNumber: Option[String] = None,
-  AdFree: Option[Boolean] = None,
-  RecurringContributionPaymentPlan: Option[String] = None,
-  MembershipJoinDate: Option[LocalDate] = None,
-  DigitalSubscriptionExpiryDate: Option[LocalDate] = None,
-  TTLTimestamp: Option[Long] = None
-) {
-
-  require(UserId.nonEmpty)
-
+case class Attributes(UserId: String,
```

Comment:
generally I stick with the 2 space indentation (and start the fields on the next line), because otherwise the amount of indentation required would change if you renamed the class, also then we will be consistent with the `(...)` indentation as the `{..}` indentation